### ISSUE TITLE
Abstract should be more like WCAG 2 abstract

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/silver#97]
From https://www.w3.org/2002/09/wbs/94845/Silver-ED-21-01-2020/results#xq2

Not sure if this is placeholder text but as-is it's not as clear, compared to say the 2.1 abstract (https://www.w3.org/TR/WCAG21/#abstract) which explains more about the aims, the technology it applies to, and how conformance relates to older versions of WCAG. I'd like the abstract to explain up front whether there are any new requirements or whether this is only a restructuring of WCAG 2.1

I'd probably go with something like "The Silver Accessibility Guidelines are the next major version update that is @@intended to supercede@@ the Web Content Accessibility Guidelines 2.x."
The W3C doesn't "replace" standards as such, it adds & supersede.

Agree with Aidan Tierney’s comments – the Silver Abstract could look at the WCAG 2.1 Abstract (https://www.w3.org/TR/WCAG21/#abstract) and be more clear about what it covers.

**Comment by jspellman** [comment]
done in FPWD
